pkg/kubeapiserver/options/plugins.go

```diff
@@ -141,6 +143,7 @@ func RegisterAllAdmissionPlugins(plugins *admission.Plugins) {
 	certsigning.Register(plugins)
 	ctbattest.Register(plugins)
 	certsubjectrestriction.Register(plugins)
+	vacsetdefault.Register(plugins)
```

Comment:
if the feature-gate is enabled,  should  the plugin be default on or off in the `DefaultOffAdmissionPlugins` func?

Reply:
If the feature-gate is enabled, the plugin be default off, similar to current storageclass flow.